### ISSUE TITLE
When I setContentView, it throw a NullPointerException

### ISSUE THREAD

**Issue by zjmok** [leanh215/PhotoPolygonCropper#1]
part of activity
```
setContentView(R.layout.activity_crop);
```
part of activity_crop:
```
    <vn.nano.photocropper.CropImageView
        android:id="@+id/img_photo"
        android:layout_width="0dp"
        android:layout_weight="1"
        android:layout_height="match_parent"
        app:layout_constraintBottom_toBottomOf="parent"
        app:layout_constraintLeft_toLeftOf="parent"
        app:layout_constraintRight_toLeftOf="@id/control"
        app:layout_constraintTop_toTopOf="parent">
    </vn.nano.photocropper.CropImageView>
```
I have not done anything else.
```
java.lang.NullPointerException: Attempt to invoke virtual method 'int android.graphics.Bitmap.getWidth()' on a null object reference
                                                                                 at vn.nano.photocropper.CropOverlayView.resetPoints(CropOverlayView.java:80)
```

**Comment by leanh215** [comment]
Let me check it

**Comment by leanh215** [comment]
I forgot to check if given bitmap == null.
Fixed and updated to version 0.0.8
